packages/webapp/src/components/Form/Unit/unit.module.scss

```diff
@@ -149,3 +149,15 @@
   transform: translate(4px, 3px);
   margin-left: 4px;
 }
+
+.currency {
+  font-size: 16px;
+  line-height: 20px;
+  color: var(--labels);
+  font-style: normal;
+  font-weight: normal;
+  font-family: 'Open Sans', 'SansSerif', serif;
+  position: absolute;
+  left: 0;
+  transform: translate(8px, 13px);
```

Comment:
To fix the slight vertical offset issue you could use 13.5px here. Alternatively (and I think preferably), since the currency and the input are in a flex container, you could add `align-self: center` here and make this `transform: translateX(8px)` only; that would also line up the symbol and the number and I think it would be a bit more robust (but a bit less similar to Input).